packages/devtools_app/lib/src/screens/performance/timeline_streams.dart

```diff
@@ -138,7 +137,7 @@ class TimelineStreamManager extends Disposer {
         service.onTimelineEvent.listen(handleTimelineEvent));
     unawaited(connectedApp.initialized.future.then((_) async {
       // The timeline is not supported for web applications.
-      if (!connectedApp.isDartWebAppNow) {
+      if (!connectedApp.isDartWebAppNow!) {
```

Comment:
what if isDartWebAppNow is null?

Reply:
Since we only call this after the `connectedApp.initialized` completes, we know it will not be null.